treezor_api_spec.yaml

```diff
@@ -12848,7 +12848,7 @@ definitions:
         type: string
       payinStatus:
         type: string
-        enum: ['PENDING', 'CANCELED', 'VALIDATED']
+        enum: ['PENDING', 'CANCELED', 'CANCELLED', 'VALIDATED']
```

Comment:
doh! ;)

Reply:
In the official open api spec, they have `CANCELED` as value. irl they send `CANCELLED` don’t ask me why

Reply:
I opened a ticket obviously, but I don’t think they will do anything about it